public/stylesheets/ie-old.min.css

```diff
@@ -1 +1 @@
ght:-20px;padding-left:20px;padding-right:20px}.membership-promo .block__title{color:inherit;font-size:28px;line-height:32px}.membership-promo .block__info{font-size:16px;line-height:20px;font-weight:900;padding-bottom:18px;font-size:21px;line-height:25px}.membership-promo .block__info .action{color:#4bc6df}.membership-promo .block__footer{position:absolute;bottom:30px}.membership-promo .button{color:#005689;background-color:#fff;border-color:#fff}.membership-promo .button:hover,.membership-promo .button:focus,.membership-promo .button:active{background-color:#f1f1f1;border-color:#f1f1f1}.package{background-color:#f1f1f1;margin-bottom:12px;position:relative}.package__list{margin-bottom:24px;margin-top:36px}.package__title{font-size:24px;line-height:28px;font-weight:900}.package__select{margin-top:3px;margin-left:5px;position:absolute;top:15px;left:10px;left:20px}.package__select:checked+.package__info{background-color:#84be43;color:#fff}.package__info{cursor:pointer;display:block;padding:12px 20px 12px 40px;padding-left:60px}.package__price{display:block;float:right;font-size:20px;line-height:24px}.package__description,.package__monthly{display:block}.button--m25{margin-bottom:24px;float:right;margin-right:0;margin-top:2px}.steps{counter-reset:li;margin-bottom:12px}.steps__item{margin-bottom:6px;min-height:46px;padding-left:50px;padding-right:10px;position:relative;width:33%;vertical-align:top;min-height:66px;padding-left:70px}.steps__item:before{background-color:#005689;color:#fff;content:counter(li);counter-increment:li;display:block;float:left;font-weight:500;line-height:40px;text-align:center;position:absolute;top:0;left:0;-webkit-border-radius:4rem;border-radius:4rem;height:40px;width:40px;font-size:32px;line-height:60px;-webkit-border-radius:6rem;border-radius:6rem;height:60px;width:60px}.digital-cta{padding-top:24px;padding-bottom:36px;padding-top:12px;padding-bottom:48px}.digital-cta__price{font-size:21px;line-height:27px;text-align:center;text-align:right}.digital-cta__amount{font-size:42px}.digital-cta__payment{color:#767676;text-align:center;text-align:right}.digital-cta__button{margin-top:14px;text-align:center;text-align:left}.basket{padding-top:2px;border-top:1px solid #005689}.basket__detail{border-top:1px dotted #dfdfdf;display:block;margin-top:6px;padding-top:8px}.basket__header{font-weight:900}.basket__smallprint{font-size:13px;line-height:18px;font-family:"Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;margin-bottom:12px}.review-details--bg{background:#f6f6f6;border-top:1px solid #dfdfdf;padding:10px}.review-details__heading{font-size:16px;line-height:24px;font-family:"Guardian Text Egyptian Web",Georgia,serif;font-weight:bold;float:left;width:140px;text-align:right}.review-details__list{margin-bottom:12px;margin-left:160px}.review-details__detail{display:inline-block;width:140px}.row--items-1 .row__item{width:100%}.global-header,.global-footer{min-width:1020px}.gs-container{min-width:980px}
```

Comment:
is this a generated file? if so, can it be generated at build time and not be present in VCS?